app/src/main/java/com/tistory/dividendcalendar/presentation/main/MainActivity.kt

```diff
@@ -170,8 +179,8 @@ class MainActivity : BaseActivity<ActivityMainBinding>(R.layout.activity_main) {
         }
     }
 
-    private fun addSettingFragment() {
-        settingFragment = SettingFragment.newInstance().also {
+    private fun addnoticeFragment() {
```

Comment:
```suggestion
    private fun addNoticeFragment() {
```

여기도 오타인 것 같아서 제안드립니다!

Reply:
오타 까지 찾아주는!! 감사합니다.

cf9df35